picard/formats/asf.py

```diff
@@ -228,7 +228,9 @@ def _remove_deleted_tags(self, metadata, tags):
                     del tags[real_name]
 
     def supports_tag(self, name):
-        return name in self.__TRANS
+        return (name in self.__TRANS
+                or name in ('~rating', '~length', 'totaldiscs', 'totaltracks')
```

Comment:
Yeah, checked WMP specification. Removed totaltracks from delete method as well in PR #541